test/functional/test_framework/key.py

```diff
@@ -380,6 +400,26 @@ def get_bytes(self):
         assert(self.valid)
         return self.secret.to_bytes(32, 'big')
 
+    def add(self, data):
+        """Add key to a scalar"""
+        secret = int.from_bytes(data, 'big')
```

Comment:
I don't think the variable name `secret` is appropriate here. Perhaps `summand`?